.github/workflows/ci-devcontainer.yaml

```diff
@@ -40,7 +40,7 @@ jobs:
 
       - name: "Install devcontainer CLI"
         id: install_devcontainer_cli
-        run: npm install -g @devcontainers/cli
+        run: npm install -g @devcontainers/cli@0.268.0
```

Comment:
Latest is `0.30.0` https://www.npmjs.com/package/@devcontainers/cli?activeTab=versions

Reply:
thank you for spotting that i was mistakenly looking at : https://www.npmjs.com/package/@vscode/dev-container-cli

- i will change it now